### PR TITLE
fmt(cstyle): cstyle fixes in src/assert.h

### PR DESCRIPTION
**Pull request by bovem**
Removed "preprocessor statement not in column 1" error

### CODE REVIEW

**Review by payes — APPROVED** [review]

File: src/assert.h [hunk 1]
```diff
@@ -106,11 +106,11 @@ do {									\
 #define	IMPLY(A, B) \
 	((void)(((!(A)) || (B)) || \
 	    REPLICA_ERRLOG("(" %s ") implies (" %s ")", \
-	    #A, #B)))
```
Comment (vishnuitta) [inline src/assert.h:109]: can you please make this as:
```
 	    REPLICA_ERRLOG("(" %s ") implies (" %s ")", #A, #B)))
```
same for below one also. Thanks

**Review by vishnuitta — CHANGES_REQUESTED** [review]
changes requested as provided in comments

**Comment by bovem** [comment]
@vishnuitta I have done the changes you requested

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**